_layouts/page.html

```diff
@@ -25,11 +25,18 @@ <h1 style="margin-top:40px" id="title">{{ page.title }}</h1>
 }
 
 function getLinks(string) {
-  const imgRex = /href="(.*?)"/g;
+  // No longer used - first strategy to just get link content
+  //const imgRex = /href="(.*?)"/g;
```

Comment:
If we’re no longer using the code, do we need to keep it around?

Reply:
Doesn’t hurt since figuring out these regex is a pain - if a future someone needs to think back it’s easily there.